content/20.treatments.md

```diff
@@ -166,6 +166,82 @@ A list of current studies and their results, using carefully the information req
+This study also treated six patients with both HCQ + azithromycin and found this combination therapy to be more effective than HCQ alone.
+This study showed design and analysis weaknesses that severely limit interpretability of results.
+These weaknesses include: lack of randomization, lack of blinding, lack of placebo, lack of Intention-To-Treat analysis, lack of correction for sequential multiple comparisons, trial arms entirely confounded by hospital, false negatives in outcome measurements, lack of trial pre-registration, and small sample size.
+Two of these issues are due to inappropriate data analysis and can therefore be corrected post-hoc by recalculating p-values (lack of Intention-To-Treat analysis and multiple comparisons.)
+However, all other issues are fundamental design flaws and can not be corrected for.
+Thus, conclusions cannot be generalized outside of the study.
+Nevertheless, this study does provide promising preliminary data on the use of HCQ in COVID-19 treatment and supports further investigation of HCQ in large, rigorous clinical trials.
+
+A randomized, non-placebo trial of 62 COVID-19 patients at the Renmin Hospital of Wuhan University studied whether HCQ decreased time to fever break or time to cough relief when compared to standard care [@doi:10.1101/2020.03.22.20040758].
+This trial found HCQ decreased both time to fever break and time to cough relief.
+However, this study also showed design and analysis weaknesses that disallow general interpretation of results.
+These weaknesses include: lack of placebo, inappropriate choice of outcomes, lack of correction for multiple primary outcomes, lack of sufficient detail to understand analysis, drastic disparities between pre-registration and published protocol, and small sample size.
+It is important to note here that the authors claim "neither the research performers nor the patients were aware of the treatment assignments."
+However, this seems impossible in a non-placebo trial - at the very least, providers would know whether they were giving HCQ or nothing, and this knowledge could lead to systematic differences in how care is given.
+Correction for multiple primary outcomes can be adjusted post-hoc by recalculating p-values, but all other issues are design and statistical problems that cannot be corrected for.
+Additionally, the issue of drastic disparities between pre-registration and published protocol may be suggestive of p-hacking.
+Again, conclusions cannot de generalized outside of the study, but the results support further investigation.
+
+A randomized trial from the Shanghai Public Health Clinical Center of 30 COVID-19 patients studied whether HCQ increased rates of virological clearance by respiratory pharyngeal swab on Day 7 post-treatment compared to standard care [@doi:10.3785/j.issn.1008-9292.2020.03.03].
```

Comment:
```suggestion
A randomized trial from the Shanghai Public Health Clinical Center of 30 COVID-19 patients studied whether HCQ increased rates of virological clearance by respiratory pharyngeal swab on Day 7 post-treatment compared to standard care [@doi:10/drbx].
```
Manubot didn't like this one, not sure why -- let's try the short DOI and see if it helps?

Reply:
I think this error is more complicated.  I tried the original and short DOI with Manubot locally and got
```
Error in get_doi_csl_item_crosscite for 10.3785/j.issn.1008-9292.2020.03.03 due to a ConnectionError:
HTTPConnectionPool(host='data.chinadoi.cn', port=80): Max retries exceeded with url: /10.3785%2Fj.issn.1008-9292.2020.03.03 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001F5D0A91320>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',))
```

Let's leave the original DOI.  I created an issue to create a manual reference for this article #148.